doc/FAQ.org

```diff
@@ -268,8 +268,13 @@ You can modify the syntax table of the mode in question. For example, for Python
 mode:
 
 #+BEGIN_SRC emacs-lisp
-(with-eval-after-load 'python
-  (modify-syntax-entry ?_ "w" python-mode-syntax-table))
+;; In your dotspacemacs/user-init()
```

Comment:
:ok_hand: 

About placing it on `dotspacemacs/user-config` I tried and couldn't make it work but my understanding of emacs and spacemacs is very small so didn't understand why :(